app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+file_name = input("Enter file name: ")
+with open(f"{file_name}.txt", "a") as f:
```

Comment:
```suggestion
with open(f"{file_name}.txt", "a") as file:
```

Reply:
This time should be good)
Could you also please check my pull requests to cache-decorator and car-wash-station?